src/components/Banner/Banner.jsx

```diff
@@ -132,36 +131,17 @@ export class Banner extends Component {
     const onCloseAlert = showClose && storage ? this.dismiss : undefined;
 
     return (
-      <div
-        className={classnames(
-          'usa-alert-full-width',
-          'vads-u-border-top--5px',
-          'medium-screen:vads-u-border-top--10px',
-          'vads-c-emergency-banner',
-          {
-            // 'info', primary-alt border, black circled 'i'
-            'vads-u-border-color--primary-alt': type === 'info' || !type,
-            // 'error', Red border, red circled exclamation
-            'vads-u-border-color--secondary': type === 'error',
-            // 'success', Green border, green checkmark
-            // 'continue', Green border, green lock
-            'vads-u-border-color--green-light':
-              type === 'success' || type === 'continue',
-            // 'warning', Yellow border, black triangle exclamation
-            'vads-u-border-color--warning-message': type === 'warning',
-          },
-        )}
-        data-e2e-id="emergency-banner"
-        onClick={this.onClick}
-      >
-        <AlertBox
-          // eslint-disable-next-line react/no-danger
-          content={<div dangerouslySetInnerHTML={{ __html: content }} />}
-          headline={title}
-          isVisible
+      <div data-e2e-id="emergency-banner" onClick={this.onClick}>
+        <VaAlert
+          visible
+          full-width
+          closeable={showClose}
           onCloseAlert={onCloseAlert}
           status={type}
-        />
+        >
+          <h3 slot="headline">{title}</h3>
+          <div dangerouslySetInnerHTML={{ __html: content }} />
```

Comment:
Mmm. :nauseated_face:  (I know it's the same as before.)

Reply:
Yeah, I had to keep myself in scope for this :sweat_smile: